pkg/river/ast/ast.go

```diff
@@ -280,7 +280,7 @@ func EndPos(n Node) token.Pos {
 	case *Comment:
 		return n.Start.Add(len(n.Text) - 1)
 	case *AttributeStmt:
-		return EndPos(n.Name)
+		return EndPos(n.Value)
```

Comment:
This was wrong, an attribute's ending position is the end of the value, not the end of the name (i.e., `attr = true` ends at `true`, not at `attr`)